app/transformers/api/v3/facility_transformer.rb

```diff
@@ -10,6 +10,7 @@ def to_response(facility)
           "teleconsultation_phone_numbers")
         .merge(config: {enable_diabetes_management: facility.enable_diabetes_management,
                         enable_teleconsultation: facility.enable_teleconsultation},
+               sync_group_id: facility.zone,
```

Comment:
Let's release this PR with `sync_group_id: facility.facility_group_id` so we can transparently add this field to the API right away.